### ISSUE TITLE
Compatibility with Playnite versions 9.3≥ 

### ISSUE THREAD

**Issue by Pherrera0** [mmatyas/pegasus-frontend#1032]
### The issue

TL;DR: The Playnite source/provider service does not work with Playnite versions 9.3≥  (current version is 10.9)

I love Pegasus and think it's the best looking front-end out there. That said, it's much easier to add games to Playnite. I was excited to see that Pegasus offered an option to import titles from Playnite. Unfortunately, it didn't work for me when I tried importing my Playnite titles. I did some digging and saw that Playnite support was added around September of last year. I downloaded the corresponding Playnite version that existed around the time that Playnite support was added, and viola, it worked! But Playnite has seen a lot of development since then, with many new (and useful) features having been added since then--so I wanted to see why Pegasus didn't recognize the new version of Playnite (I thought it might be something I could fix on my own, like files being kept in a different directory). 

Turns out that Pegasus looks for .json files containing game info in `%AppData%\playnite\library\games`; which is as intended, as Playnite versions preceeding 9.3 stored game info in distinct .json files under this directory (e.g., `%AppData%\playnite\library\games\1ef635d7-5448-43bb-a098-72fca52aea68.json`)

However, in version 9.3 (and onward), game info is stored in a single .db file located at `%AppData%\playnite\library` (e.g.,  `%AppData%\playnite\library\games.db`). I opened this .db file in notepad, and it looks like it's a liteDB file and it seems like it has all the same info as the last database.

### Playnite `%AppData%\playnite\` archives

I've attached archives of `%AppData%\playnite\` for both versions. Pegasus works with the database in version 8.15, but versions 9.3 onward use a different database (I've attached an archive for 10.9 since it's the current version as of writing this). I think up-to-date Playnite compatibility could really be a game-changer for Steam Deck (Windows) owners. Please let me know if there's anything else you need. 

[Playnite 10.9 appdata roaming.zip](https://github.com/mmatyas/pegasus-frontend/files/10290815/Playnite.10.9.appdata.roaming.zip)
[Playnite 8.15 appdata roaming.zip](https://github.com/mmatyas/pegasus-frontend/files/10290816/Playnite.8.15.appdata.roaming.zip)

### System info

Steam Deck running Windows 10 22H2. 

### Pegasus version

alpha16-44-g8119fcec, 2022-10-14

### Pegasus log

[lastrun.log](https://github.com/mmatyas/pegasus-frontend/files/10290771/lastrun.log)

**Comment by mmatyas** [comment]
Thank you very much for the detailed description! I'll see if I can update the Playnite support after the holidays.

**Comment by Daisouji** [comment]
I think it would be better to have a variable for the location of that file instead of just looking through appdata.

**Comment by Daisouji** [comment]
Is this issue being worked on?

**Comment by Kenoux** [comment]
Hi All,
Any news about fixing Playnite into Pegasus ?

Thanks for the work
